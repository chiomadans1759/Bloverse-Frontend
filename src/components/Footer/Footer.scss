@import url('https://fonts.googleapis.com/css?family=Comfortaa');
.footer-container {
  margin: auto;
  width: 100%;
  bottom: 0;
  text-align: center;
  padding: rem(10px);
  background: transparent
}

.bloverse {
  font-size: rem(16px);
  font-weight: 900;
  font-family: "Comfortaa" !important;
}
