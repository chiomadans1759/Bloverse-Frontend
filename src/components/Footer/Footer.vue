<template>
  <div class="footer-container">
    <p><span class="bloverse">Bloverse</span> {{year}}. All rights reserved</p>
  </div>
</template>

<script>
export default {
  data() {
    return {
      year: new Date().getFullYear()
    }
  }
}
</script>


<style lang="scss">
@import "./Footer.scss";
</style>
