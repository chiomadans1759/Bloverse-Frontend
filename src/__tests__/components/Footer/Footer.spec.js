import { mount } from '@vue/test-utils';
import Footer from '@/components/Footer/Footer.vue';

describe('Footer Component', () => {
  it('should render correctly', () => {
    const wrapper = mount(Footer);

    expect(wrapper.element).toMatchSnapshot();
  });
});
