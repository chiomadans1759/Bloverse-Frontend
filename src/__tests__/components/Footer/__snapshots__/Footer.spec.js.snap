// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer Component should render correctly 1`] = `
<div
  class="footer-container"
>
  <p>
    <span
      class="bloverse"
    >
      Bloverse
    </span>
     2019. All rights reserved
  </p>
</div>
`;
